packages/grafana-ui/src/components/Layout/Grid/GridItem.tsx

```diff
@@ -0,0 +1,57 @@
+import { css } from '@emotion/css';
+import React, { useCallback } from 'react';
+
+import { useStyles2 } from '../../../themes';
+
+export interface GridItemProps {
```

Comment:
There are no props for further styling yet (e.g. background colour, font size or colour, etc.) which forces users of GridItem to create another div inside. Is this something we can add or is the intention here to keep the component as basic as possible?

Reply:
We want to keep it as basic as possible. Also, when using GridItem we are setting `display: contents`. It behaves like it does not occupy any space so the element to have the styling is the child itself.